src/screens/GroupInfo/elements/Media.js

```diff
@@ -38,7 +39,7 @@ export default Media;
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: '#fff',
+    backgroundColor: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing hard-coded color values with references to color constants from a theme file improves maintainability and consistency of the code.
```diff
-    backgroundColor: '#fff',
+    backgroundColor: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->